nvflare/private/fed/client/client_runner.py

```diff
@@ -327,50 +312,38 @@ def _do_process_task(self, task: TaskAssignment, fl_ctx: FLContext, abort_signal
         self.log_debug(fl_ctx, "firing event EventType.BEFORE_TASK_RESULT_FILTER")
         self.fire_event(EventType.BEFORE_TASK_RESULT_FILTER, fl_ctx)
 
-        filter_list = []
-        if scope_object and scope_object.task_result_filters:
-            filter_list.extend(scope_object.task_result_filters)
-
-        task_filter_list = self.task_result_filters.get(task.name)
-        if task_filter_list:
-            filter_list.extend(task_filter_list)
-
-        if filter_list:
-            for f in filter_list:
-                filter_name = f.__class__.__name__
-                try:
-                    reply = f.process(reply, fl_ctx)
-                except UnsafeJobError:
-                    self.log_exception(fl_ctx, f"UnsafeJobError from Task Result Filter {filter_name}")
-                    executor.unsafe = True
-                    fl_ctx.set_job_is_unsafe()
-                    return self._reply_and_audit(
-                        reply=make_reply(ReturnCode.UNSAFE_JOB),
-                        ref=server_audit_event_id,
-                        fl_ctx=fl_ctx,
-                        msg=f"submit result: {ReturnCode.UNSAFE_JOB}",
-                    )
-                except Exception as e:
-                    self.log_exception(
-                        fl_ctx, f"Processing error in Task Result Filter {filter_name}: {secure_format_exception(e)}"
-                    )
-                    return self._reply_and_audit(
-                        reply=make_reply(ReturnCode.TASK_RESULT_FILTER_ERROR),
-                        ref=server_audit_event_id,
-                        fl_ctx=fl_ctx,
-                        msg=f"submit result: {ReturnCode.TASK_RESULT_FILTER_ERROR}",
-                    )
+        try:
+            filter_name = f"{Scope().task_result_filters=}".split("=")[0].split(".")[-1]
```

Comment:
This line might be hard to read.

Essentially it is trying to get ```filter_name = "task_result_filters"```

Should we just hard-coded it? or use some constants variable in Scope.py?

Reply:
I thought about this before. It's not good to hard-coded, or use constant for this. If the variable name got renamed, then it's likely this will be missed and broke it.